src/Pester.RSpec.ps1

```diff
@@ -363,13 +363,13 @@ function New-PesterConfiguration {
       Default value: $true
 
     TestResult:
-      Enabled: Enable TestResult.
+      Enabled: Enable export of test results to a file.
       Default value: $false
 
       OutputFormat: Format to use for test result report. Possible values: NUnitXml, NUnit2.5 or JUnitXml
       Default value: 'NUnitXml'
 
-      OutputPath: Path relative to the current directory where test result report is saved.
+      OutputPath: Path relative to the current directory where test result report is saved. Enabled has to be set to true to have test results exported
```

Comment:
```suggestion
      OutputPath: Path relative to the current directory where test result report is saved. Enabled has to be set to $true to have test results exported.
```

But maybe it would be better to write this next to the first option? It is on the top, and you don't need to repeat this on every option in this section.